### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by Biohazord**
Updated readme example to reflect functionality.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -53,11 +53,12 @@ $ npm install --save-dev backbone.react-bridge
       // Customize the form of the properties which will be passed to the
       // React Component. In case that 'getProps' is undefined, a composition
       // of the model's attributes, the collection's values and the custom
-      // properties will be returned to the React Component
+      // properties will be returned to the React Component.
+      // `getProps` receives an object with `model` and `collection` as members.
```
Comment (imktks) [inline README.md]: Nit: ...receives an object with `model` and `collection` as **properties**

**Comment by imktks** [comment]
Github Tip: Add `Resolves #4` in the commit message description in order to automatically close the corresponding Github issue.

https://help.github.com/articles/closing-issues-using-keywords/

File: README.md (hunk 1 above)
Comment (Biohazord) [reply README.md]: Updated

**Comment by Biohazord** [comment]
There are additional properties passed in. Are those also worth mentioning?  I also see `getProps`, `props` and `observe`. The only thing I can see that may be useful would be `props`. 

I also forgot to add that comment to my commit message.

**Pull request merged**